main/build.gradle

```diff
@@ -337,6 +337,10 @@ dependencies {
     // GeographicLib
     implementation 'net.sf.geographiclib:GeographicLib-Java:2.0'
 
+    // DB inspection
+    implementation 'com.github.moving-bits:datatables:-SNAPSHOT'
+    implementation 'com.github.moving-bits:dbinspection:b6c15506ee'
```

Comment:
General question to @cgeo/team.
Is it really necessary to have separate external projects here? Wouldn't it be better to have them in c:geo (directly or as dedicated repos)?

Reply:
If we want to go this path I vote for direct code integration . Except if database inspector is maintained and published as it's own library to a public place like maven central, then of course we can just link it.

Reply:
That's what this library is intended to be (as it's use case goes beyond c:geo). It's a public repo and the lib is published to Jitpack already.